client/src/components/BookEvent/index.js

```diff
@@ -15,125 +45,154 @@ class BookEvent extends React.Component {
+                    name="start_date"
+                  />
+
+                  <input
+                    className="note"
+                    type="text"
+                    name="note"
+                    placeholder="Note:"
+                    value={this.state.note}
+                    onChange={this.handleChange}
+                  />
+                  <input
+                    className="input"
+                    type="text"
+                    name="capacity"
+                    placeholder="#Persons:"
+                    value={this.state.capacity}
+                    onChange={this.handleChange}
+                    required
+                  />
+                  <h2 className="h2">Lunch</h2>
+                  <input
+                    className="note"
+                    type="text"
+                    name="lunch_note"
+                    placeholder="Description:"
+                    value={this.state.lunchnote}
+                    onChange={this.handleChange}
+                  />
+
+                  <h2 className="coffee">Coffee</h2>
+                  <input
+                    className="note"
+                    type="text"
+                    name="coffee_note"
+                    placeholder="Description:"
+                    value={this.state.coffeenote}
+                    onChange={this.handleChange}
+                  />
+
+                  <h2 className="equipment">Equipment</h2>
+                  <input
+                    className="note"
+                    type="text"
+                    name="equipment_note"
+                    placeholder="Description:"
+                    value={this.state.equipmentnote}
+                    onChange={this.handleChange}
+                  />
+                </div>
+                <div className="column2">
+                  <input
+                    className="input"
+                    type="text"
+                    name="org_name"
+                    placeholder="Organization Name"
+                    value={this.state.orgName}
+                    onChange={this.handleChange}
+                    required
+                  />
+                  <label className="label">End Date:</label>
+                  <DatePicker
+                    className="bookdates"
+                    selected={this.state.end_date}
+                    onChange={this.handleChangeEndDate}
+                  />
+
+                  <input
+                    className="input"
+                    type="Number"
+                    name="price"
+                    placeholder="Cost:"
+                    value={this.state.price}
+                    onChange={this.handleChange}
+                  />
+                  <input
+                    className="lunchcost"
+                    type="Number"
+                    name="lunch_price"
+                    placeholder="Cost/Person:"
+                    value={this.state.lunchprice}
```

Comment:
same thing here ```lunchPrice``` is more readable than ```lunchprice``` and for all other variables

Reply:
done